cmake/Config.cmake.in

```diff
@@ -30,9 +30,22 @@ if(@OPENMP_FOUND@) # Indicates if OpenMP has been used when compiling the Radium
     find_dependency(OpenMP REQUIRED)
 endif()
 # Theses paths reflect the paths founds in RadiumEngine/src/Core/external/package
-set(Eigen3_DIR    "${CMAKE_CURRENT_LIST_DIR}/../../../@Eigen3_sub_DIR@")
-set(OpenMesh_DIR  "${CMAKE_CURRENT_LIST_DIR}/../../../@OpenMesh_sub_DIR@")
-set(cpplocate_DIR "${CMAKE_CURRENT_LIST_DIR}/../../../@cpplocate_sub_DIR@")
+if ("@Eigen3_DIR@" STREQUAL "" )
```

Comment:
Is there any way to automate this instead of having the information in two places ?

Reply:
I think we already automated this a lot... the problem is that we need to forward information from config; build and install times, for both Radium and the externals.